src/features/device-state/routes/state-get-v2.ts

```diff
@@ -58,8 +58,8 @@ export type StateV2 = {
 		};
 	};
 	dependent: {
-		apps: {};
-		devices: {};
+		apps: object;
+		devices: object;
```

Comment:
Fwiw I think these should be specifically an empty object, I don't know off the top of my head how you type that?